wget: unable to resolve host address ‘ftp.cixug.es’

recently, when i run make PythonAPI, i encounter wget issue, 
--2022-03-30 18:30:20--  https://ftp.cixug.es/apache//xerces/c/3/sources/xerces-c-3.2.3.tar.gz
Resolving ftp.cixug.es (ftp.cixug.es)... failed: Name or service not known.
wget: unable to resolve host address ‘ftp.cixug.es’
make: *** [Util/BuildTools/Linux.mk:137: setup] Error 4

can someone help me out?

--- a/Util/BuildTools/Setup.sh
+++ b/Util/BuildTools/Setup.sh
@@ -429,7 +429,7 @@ XERCESC_VERSION=3.2.3
 XERCESC_BASENAME=xerces-c-${XERCESC_VERSION}
 
 XERCESC_TEMP_FOLDER=${XERCESC_BASENAME}
-XERCESC_REPO=https://ftp.cixug.es/apache//xerces/c/3/sources/xerces-c-${XERCESC_VERSION}.tar.gz
+XERCESC_REPO=https://downloads.apache.org/xerces/c/3/sources/xerces-c-${XERCESC_VERSION}.tar.gz
 
 XERCESC_SRC_DIR=${XERCESC_BASENAME}-source
 XERCESC_INSTALL_DIR=${XERCESC_BASENAME}-install

